### ISSUE TITLE
Coaxial gear has broken render

### ISSUE THREAD

**Issue by NthPortal** [mrsterner/CreateCogglewoggle#4]
The coaxial gear does not render correctly, as can be seen in the following screenshot:

![2024-09-11_00 20 57](https://github.com/user-attachments/assets/e238989a-bbb6-4d0e-a6df-aca42a0502bc)

- The shaft sharing a block with the coaxial gear turns with the gear rather than with the attached shaft.
- When you attach a shaft to the side of the coaxial gear that does not already have an incoming shaft, it creates a false shaftless cogwheel *render* replacing the shaft *render* immediately on the other side of the coaxial gear; however, the shaft is still there and seems to behave properly. The cogwheel render appears to rotate at the same speed as the shaft it actually is, and ignores the rotation of other cogwheels that it appears to mesh with.

In summary, the coaxial gear seems to *behave* fine but render quite incorrectly.
